definitions/contested/json/CaseEventToFields/CaseEventToFields-shareACase-nonprod.json

```diff
@@ -18,7 +18,7 @@
     "CaseEventID": "FR_newPaperCase",
     "CaseFieldID": "ApplicantOrganisationPolicy",
     "PageFieldDisplayOrder": 16,
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "COMPLEX",
```

Comment:
needs to add this event FR_newPaperCase to CaseEventToComplexType, alike FR_solicitorCreate, if we change the type to COMPLEX